WordPress/UITestsFoundation/Screens/Editor/BlockEditorScreen.swift

```diff
@@ -257,21 +257,26 @@ public class BlockEditorScreen: ScreenObject {
     public func postAndViewEpilogue(action: postAction) throws -> EditorPublishEpilogueScreen {
         try post(action: action)
         waitAndTap(noticeViewButton)
+
         return try EditorPublishEpilogueScreen()
     }
 
     public func post(action: postAction) throws {
         let postButton = app.buttons[action.rawValue]
         let postNowButton = app.buttons["\(action.rawValue) Now"]
         var tries = 0
+
         // This loop to check for Publish/Schedule Now Button is an attempt to confirm that the postButton.tap() call took effect.
         // The tests would fail sometimes in the pipeline with no apparent reason.
         repeat {
             postButton.tap()
             tries += 1
         } while !postNowButton.waitForIsHittable(timeout: 3) && tries <= 3
 
-        try confirmPost(button: postNowButton, action: action)
+        postNowButton.tap()
+        guard action == .publish else { return }
+
+        dismissBloggingRemindersAlertIfNeeded()
```

Comment:
Not blocking in any way, just curious what is the reason for moving this code here and getting rid of `confirmPost` function?

Reply:
thanks for the review! so the reason is that i saw that it was only used in one place and considering this test is a bit flaky, i was looking for ways to make it "leaner" by not breaking all the methods into smaller ones (also i find it easier to read instead of having to jump to multiple methods to know what it is doing). however, if this doesn't help or makes it worse, we can revert the changes too.